grafana/mattermost-performance-monitoring-v2.json

```diff
@@ -2608,12 +2858,13 @@
     ]
   },
   "time": {
-    "from": "now-6h",
+    "from": "now-15m",
     "to": "now"
   },
   "timepicker": {},
   "timezone": "",
   "title": "Mattermost Performance Monitoring v2",
-  "uid": "im7xNX17k",
-  "version": 4
-}
+  "uid": "im7xNX17kdd",
+  "version": 1,
+  "weekStart": ""
```

Comment:
Should the ID change? I'd only expect a version bump.

Reply:
The id can change, that's fine.